tests/integration/adapters/test_redis_postgresqllobe_db.py

```diff
@@ -0,0 +1,93 @@
+import psycopg2
+import pytest
+
+from hyperon_das_atomdb.adapters import RedisPostgreSQLLobeDB
+from hyperon_das_atomdb.database import WILDCARD
+
+from .helpers import Database, _db_down, _db_up, cleanup, postgresql_port, redis_port
+
+
+class TestRedisPostgreSQLLobeDB:
+    @pytest.fixture(scope="session", autouse=True)
+    def _cleanup(self, request):
+        return cleanup(request)
+
+    def _populate_db(self):
+        conn = psycopg2.connect(
+            host='localhost',
+            database='postgres',
+            port=postgresql_port,
+            user='dbadmin',
+            password='dassecret',
+        )
+        with conn.cursor() as cursor:
+            cursor.execute(
+                """
+                CREATE TABLE IF NOT EXISTS users (
+                    ID SERIAL PRIMARY KEY,
+                    name VARCHAR(100),
+                    age INTEGER
+                );
+                """
+            )
+            cursor.execute("INSERT INTO users (name, age) VALUES ('Adam', 30);")
+            cursor.execute("INSERT INTO users (name, age) VALUES ('Eve', 31);")
```

Comment:
Shouldn't Eve be younger than Adam? ;-)

Reply:
:smile:
Yeah. But women must come first, we're nothing without them :smiley: